command/gems/gemfile_lock.go

```diff
@@ -86,3 +89,41 @@ func ParseBundlerVersion(gemfileLockContent string) (gemVersion Version, err err
 		Found:   true,
 	}, nil
 }
+
+var gemFileLockNames = []string{
+	"Gemfile.lock", "gems.locked",
+}
+
+// GemFileLockPth gets the path for the gem lock file from the given directory.
+func GemFileLockPth(searchDir string) (string, error) {
+	var pth string
+	for i, gemFileName := range gemFileLockNames {
+		pth = filepath.Join(searchDir, gemFileName)
+
+		if _, err := os.Stat(pth); err != nil && i == len(gemFileLockNames)-1 {
+			return "", err
+		} else if err == nil {
+			return pth, nil
+		}
+	}
+	return pth, nil
+}
+
+// GemFileLockContent gets the content of the gem lock file from the given directory.
+func GemFileLockContent(searchDir string) (string, error) {
```

Comment:
Not entirely convinced that GemFileLockContent and ParseVersionFromBundlePth are helpful enough to be added as a package level exported functions.

Reply:
Would be needed for fastlane step.